src/app_uart.c

```diff
@@ -405,6 +407,8 @@ rd_status_t app_uart_send_broadcast (const ri_adv_scan_t * const scan)
     {
         memcpy (adv.params.adv.mac, scan->addr, sizeof (adv.params.adv.mac));
         memcpy (adv.params.adv.adv, scan->data, scan->data_len);
+        manuf_id = scan->data[5];
```

Comment:
Nothing guarantees positioning of manufacturer ID in scanned data, at least the type `0xFF` should be checked. 

Essentially we're looking for sequence `0xFF, ID[1] ID[0]`